gce-jetty/src/main/docker/Dockerfile

```diff
@@ -27,11 +27,14 @@ RUN mv /usr/local/jetty-distribution-* $JETTY_HOME && chown -R jetty:jetty $JETT
 ENV JETTY_BASE /var/lib/jetty
 RUN mkdir -p $JETTY_BASE
 WORKDIR $JETTY_BASE
-RUN java -jar $JETTY_HOME/start.jar --add-to-startd=http,deploy,setuid && \
+RUN java -jar $JETTY_HOME/start.jar --add-to-startd=http,deploy,jsp,jstl,setuid && \
     chown -R jetty:jetty $JETTY_BASE && \
-    mkdir /tmp/jetty
+    mkdir /tmp/jetty && chown -R jetty:jetty /tmp/jetty
+
+RUN ln -s $JETTY_BASE/webapps/root /app
+WORKDIR /app
```

Comment:
I think for the pure jetty image, it is best to have the workdir as jetty_base, as derived images will have to customize that anyway. 

I'll accept and fix.